cluster/src/main/java/com/hartwig/pipeline/tertiary/orange/Orange.java

```diff
@@ -152,9 +152,13 @@ private List<BashCommand> buildCommands(final SomaticRunMetadata metadata) {
         final List<String> primaryTumorDoids = metadata.tumor().primaryTumorDoids();
         String primaryTumorDoidsString = "\"" + String.join(";", primaryTumorDoids) + "\"";
         String linxPlotDir = linxSomaticOutputDir.getLocalTargetPath() + "/plot";
+        // TODO can we determine experiment type from pipeline metadata? Can be WGS or PANEL, hardcode to WGS for now
```

Comment:
Since this "buildCommands" is only being called by tumorReferenceCommands, you have essentially already determined that this is a WGS experiment type, as we do not run targeted with both tumor and reference. But indeed that's a bit implicit.

You could use the `Arguments.useTargetRegions()` to more explicitly discriminate.

Regardless of what we decide, lets remove the TODO, we generally don't have them in this code base. You can mention in the commit message instead to have some history/rationale.